queries/typescript/highlights.scm

```diff
@@ -38,6 +38,8 @@
 
 (template_literal_type) @string
 
+"!" @operator
```

Comment:
If anything, I'd say this should also be in ecma/highlights. Don't see why only typescript is allowed to have this highlighted

Reply:
well, in ecma it's distinguished on L257:
```(unary_expression ["!" "~" "-" "+"] @operator)```

Reply:
Yeah, but I don't remember any explicit case where "!" can be used in typescript that is not in javascript tho

Reply:
```ts
const result = maybeundefinedreturn()!
```

Reply:
Well I don't know of that bizarre syntax :smile: